examples/context/forwarding-refs.js

```diff
@@ -0,0 +1,12 @@
+const Button = ({theme, children}) => (
+  <button className={theme ? 'dark' : 'light'}>
+    {children}
+  </button>
+);
+
+// highlight-range{1,3}
+export default React.forwardRef((props, ref) => (
+  <ThemeContext.Consumer>
+    {theme => <Button {...props} theme={theme} ref={ref} />}
+  </ThemeContext.Consumer>
+));
```

Comment:
This example doesn't make sense, because `Button` is a functional component. I'll update it!